function-controller/Makefile

```diff
@@ -3,33 +3,28 @@ OUTPUT = function-controller
 GO_SOURCES = $(shell find pkg cmd -type f -name '*.go')
 TAG = 0.0.5-snapshot
 
-build: $(OUTPUT) vendor
+build: $(OUTPUT)
 
-test: vendor
-	go test -v `glide novendor`
+test:
+	go test ./...
 
-$(OUTPUT): $(GO_SOURCES) vendor
+$(OUTPUT): $(GO_SOURCES)
 	go build cmd/function-controller.go
 
-vendor: glide.lock
-	glide install -v --force
-
-glide.lock: glide.yaml
-	glide up -v --force
-
-gen-mocks:
-	mockery -name 'TopicInformer|FunctionInformer' -dir vendor/github.com/projectriff/kubernetes-crds/pkg/client/informers/externalversions/projectriff/v1
-	mockery -name 'SharedIndexInformer' -dir vendor/k8s.io/client-go/tools/cache
-	mockery -name 'DeploymentInformer' -dir vendor/k8s.io/client-go/informers/extensions/v1beta1
-	mockery -name 'LagTracker|Deployer' -dir pkg/controller
+#TODO
+#gen-mocks:
```

Comment:
does this mean testing is currently not working? (wondering why there is no `test` target in the root build)

Reply:
no, just that re-generating mocks is not supported (given they involved paths inside vendor)